doozerlib/cli/images_streams.py

```diff
@@ -124,6 +124,13 @@ def images_streams_mirror(runtime, streams, only_if_missing, live_test_mode, dry
                     print(f'For {upstream_entry_name}, would have run: {arm_cmd}')
                 else:
                     exectools.cmd_assert(arm_cmd, retries=3, realtime=True, timeout=1800)
+            # mirror builder image streams to priv namespace
+            if config.upstream_image_mirror is not Missing:
+                priv_cmd = f'oc image mirror {upstream_dest} {config.upstream_image_mirror}'
+                if dry_run:
+                    print(f'For {upstream_entry_name}, would have run: {priv_cmd}')
+                else:
+                    exectools.cmd_assert(priv_cmd, retries=3, realtime=True)
```

Comment:
Looks like we might be missing the optional registry config dir in the command line we are building up: https://github.com/openshift-eng/doozer/pull/817/files#diff-01fdff6d5a9f916a747cb4715872dca3b457bcca96c7e8a278627d522adb723bR110-R111 .

Reply:
updated in https://github.com/openshift-eng/doozer/pull/817/commits/2bbe1a3cd59b5ae94d52d4d5b74238164211a593